eslint-example/package.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "eslint-example",
+    "version": "1.0.0",
+    "description": "",
+    "main": "index.js",
+    "dependencies": {},
+    "devDependencies": {
+        "eslint": "^3.9.0",
+        "eslint-plugin-mozilla": "^0.2.3"
```

Comment:
You shouldn't need `eslint-plugin-mozilla` - that's more for the old style bootstrap extensions that access the chrome privileged code.

My example repo has a [list of plugins](https://github.com/Standard8/example-webextension/blob/master/docs/Linting.md) that might be useful.